src/Database/Mongo/Results.purs

```diff
@@ -33,14 +33,23 @@ instance decodeJsonWriteResult :: DecodeJson WriteResult where
       }
 
 instance encodeJsonWriteResult :: EncodeJson WriteResult where
-  encodeJson (WriteResult w) = jsonEmptyObject
+  encodeJson (WriteResult w)
+    =  "ok"        := boolToJsNumber w.success
+    ~> "n"         := w.total
+    ~> "nInserted" := fromMaybe 0.0 w.inserted
+    ~> "nModified" := fromMaybe 0.0 w.modified
```

Comment:
Thanks :)

BTW, I have a question: I used `0.0` here instead of `0` because `WriteResult` takes `Maybe Number`, and `0` is a `Int` not a `Number`. But I don't understand why `Int` isn't a `Number`? I tried `(0 :: Number)`, but didn't typecheck, that's why I used `0.0`. The same trick I used [here](https://github.com/dicefm/purescript-node-mongodb/pull/25/files#diff-bdfce02a513a3c793bd5d15141497925R23)